src/components/App.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+import Header from './Header'
+import Prompt from './Prompt'
+
+function App() {
+  const prompts = [
+    'Big Book',
+    'Bought Recently',
+    'Classic',
+    'Cover Colour',
+    'Dark Academia',
+    'DNF',
+    'Fantasy',
+    'Feminine Protagonist',
+    'Friend Pick',
+    'Graphic Novel',
+    'Haunted Setting',
+    'Highest Rated',
+    'Lowest Rated',
+    'Masculine Protagonist',
+    'Un-Read Author',
+    'New-ish',
+    'Non-Fiction',
+    'Page Number',
+    'Romance',
+    'Short Read',
+    'Thriller',
+  ]
+
+  const [currentPrompt, setCurrentPrompt] = useState(
+    'Spin to receive your first prompt.'
+  )
+
+  return (
+    <Grid>
+      <Header>Wheel of TBR</Header>
+      <Main>
+        <Prompt>{currentPrompt}</Prompt>
+        <FlexWrapper>
+          <Button primary onClick={getPrompt}>
+            Spin!
+          </Button>
+        </FlexWrapper>
+      </Main>
+    </Grid>
+  )
+
+  function getPrompt() {
```

Comment:
Please consider a different name. *get*Something implies that the function returns a value. Maybe `setPrompt` or `setRandomPrompt`?